src/router/routes/users.ts

```diff
@@ -71,6 +71,27 @@ export class UsersController extends ParentRouter implements IExpressController
       this.authService.verifyAcl(this.extraCreditPerm, AclOperations.CREATE),
       (req, res, next) => this.addExtraCreditAssignmentHandler(req, res, next),
     );
+    app.get(
+      '/extra-credit/assignment',
+      this.authService.verifyAcl(this.aclPerm, AclOperations.READ),
+      (req, res, next) => this.getExtraCreditAssignmentHandler(req, res, next),
+    );
+    app.get(
```

Comment:
This won't work. Signifying the queries in `app.get()` isn't supported. Instead, follow the design as per the `admin/admin-statistics.ts` file and create a handler in the middle that calls these handler functions. In order to do the proper acl verification, you can call `this.authService.verifyAclRaw()` to get boolean output about the permission.

Reply:
Talk to me in person for more details on this.